src/directives/NgGrid.ts

```diff
@@ -459,15 +459,17 @@ export class NgGrid implements OnInit, DoCheck, OnDestroy {
 		this.setConfig(this._config);
 	}
 
-	private _onMouseDown(e: any): boolean {
+	private _onMouseDown(e: MouseEvent): boolean {
 		var mousePos = this._getMousePosition(e);
 		var item = this._getItemFromPosition(mousePos);
 
 		if (item != null) {
 			if (this.resizeEnable && item.canResize(e)) {
 				this._resizeReady = true;
 			} else if (this.dragEnable && item.canDrag(e)) {
-				this._dragReady = true;
+				this._dragReady = true; 
+				// Prevent default behaviour (text selection) in safari
+				return false;

```

Comment:
I think I didn't put those here intentionally. There was a separate issue caused by preventing the mouse down behaviour here, when doing a single click, for example.